core-bundle/src/Doctrine/Backup/BackupManager.php

```diff
@@ -135,9 +174,10 @@ private function doCreate(CreateConfig $config): void
         $backup = $config->getBackup();
 
         // Ensure the target file exists and is empty
-        (new Filesystem())->dumpFile($backup->getFilepath(), '');
+        $this->vfs->write($backup->getFilename(), '');
 
-        $fileHandle = fopen($backup->getFilepath(), 'w');
+        $tmpFile = (new Filesystem())->tempnam(sys_get_temp_dir(), 'ctobckupmgr');
```

Comment:
Is the idea of using this `$tmpFile` that local writes are faster in general? We could also mount a `tmp` storage then and just use `move()` when finished. But maybe I'm not fully understanding the implications. :relaxed:

Reply:
No the problem is compression and decompression. To get rid of the intermediary local temp file, I think we would need some kind of `GzFilesystem` but it needs to be optional. I mean, you can give it a try if you want to, then you see the problem 👼